.github/workflows/deploy.yml

```diff
@@ -2,11 +2,12 @@ name: Deploy Website to Github Pages
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
+  pull_request:
 
 jobs:
   build_and_deploy:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
In general I prefer and recommend using explicit host platform versions, it helps prevent random breakages when they move the "latest" platform underneath you. In this case we are downloading a specific dependency which is only built for 20.04 so we should also use that as our build platform.

Reply:
Thanks! That's a great point.